packages/shared.test/src/stubs/ontologies/legacyFullOntology.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import invariant from "tiny-invariant";
+import { FauxDataStore } from "../../FauxFoundry/FauxDataStore.js";
+import { FauxFoundry } from "../../FauxFoundry/FauxFoundry.js";
+import { FauxOntology } from "../../FauxFoundry/FauxOntology.js";
+import { registerLazyActions } from "../actions.js";
+import { ActionTypeWithUnsupportedTypes } from "../actionsTypes.js";
+import { BarInterface, FooInterface } from "../interfaces.js";
+import {
+  employee1 as employee_John_50030,
+  employee2 as employee_Jane_50031,
+  employee3,
+  employee4withDerived,
+  employee50050,
+  employeeFailsStrict,
+  nycOffice,
+  objectWithAllPropertyTypes1,
+  objectWithAllPropertyTypesEmptyEntries,
+  travisPlayer,
+} from "../objects.js";
+import { objectTypesWithLinkTypes } from "../objectTypesWithLinkTypes.js";
+import { queryTypes } from "../queryTypes.js";
+import { fooSpt } from "../spts.js";
+import { defaultOntologyMetadata } from "./defaultOntologyMetadata.js";
+
+//
+// Setup the ontology
```

Comment:
Once everything is complete (or complete enough to keep everything passing) I will refactor all of this so that you can get the legacy testing msw or create your own for really predictable tests (and complexity too!)